pkg/core/fs_rw_ops.go

```diff
@@ -536,10 +540,12 @@ func (fs *fsMutable) FlushFile(
 	f := fs.backingFiles[op.Inode]
 	if f != nil {
 		file := *f
+if file != nil {
```

Comment:
I don't quite understand when we insert a nil in the map.
I believe there is still a possible panicking cause in `SyncFile()` with `file := *backingFiles[op.Inode]` 
but `FlushFile` should be safe, calling `file.Sync()` without necessarily passing by the `file := *fs`  staging.
Like:
```go
	file := fs.backingFiles[op.Inode]
	if file != nil {
		err := file.Sync()
		if err != nil {
			return fuse.EIO
		}
	}
	return
```

Reply:
i threw this conditional in while debugging .. yes, it needs to be removed before merge.